sops/__init__.py

```diff
@@ -178,6 +178,9 @@ def main():
                            help="path to config file, disable recursive search"
                                 " (default: {default})"
                                 .format(default=DEFAULT_CONFIG_FILE))
+    argparser.add_argument('--encryption-context', dest='context',
+                           help="KMS encryption context: "
+                                "key-value pair dict encoded in JSON")
```

Comment:
I haven't used KMS encryption contexts, but would there be a way to expand this to not require JSON on the command line but take individual parameters instead? We already have a custom csv format for specifying multiple kms and pgp on the command line, and I'd prefer to avoid introducing another format.